### PR TITLE
TCG Card Shop Simulator YML

### PR DESCRIPTION
**Pull request by SaxbyMod**
TCG Card Shop Simulator

TCG Card Shop Simulator Modding - Icon [880x880]
image - Banner/R2 Icon May come back and change this later on if I can spot the place to do such at.

### CODE REVIEW

**Comment by SaxbyMod** [comment]
image
The Icon for r2 fcked up on copy sorry bout that

**Comment by SaxbyMod** [comment]
This should be a better version of the r2 banner thing
image

**Review by anttimaki — COMMENTED** [review]
This PR has some extra fields that I'm not familiar with, so I'm hesitant to review this PR. Are you sure e.g. the "other" platform is really supported by the mod manager? And are the BepInEx configs really necessary, or were they copy pasted from somewhere.

Here is a "text book example" of simple PR: #75. If this PR looked something like this, I could review it and we wouldn't need to wait for Mythic. That's of course assuming the default config for BepInEx is ok for the game. Also note that `autolistPackageIds` in that PR points BepInEx for games that use IL2CPP. If this game uses mono, the value should be `"BepInEx-BepInExPack"` instead.

Or, if you know what you're doing and believe the extra fields are necessary, we can wait for Mythic's review.

**Comment by SaxbyMod** [comment]
Should be resolved now

File: games/data/tcg-card-shop-simulator.yml
Comment (anttimaki) [inline games/data/tcg-card-shop-simulator.yml:1]: This seems to be an exact match with GoreBox. A game needs an unique uuid. Grab one e.g. from https://www.uuidgenerator.net/version4

**Review by anttimaki — CHANGES_REQUESTED** [review]

File: games/data/tcg-card-shop-simulator.yml
Comment (SaxbyMod) [reply games/data/tcg-card-shop-simulator.yml:1]: Is there a way for me to generate it while making sure it doesnt match pre existing ones and ill try and fix this around 4-5pm EST ^^

Comment (anttimaki) [reply games/data/tcg-card-shop-simulator.yml:1]: Just use a random generated one and it shouldn't be a problem.

> The number of random version-4 UUIDs which need to be generated in order to have a 50% probability of at least one collision is 2.71 quintillion. This number is equivalent to generating 1 billion UUIDs per second for about 86 years.

Comment (SaxbyMod) [reply games/data/tcg-card-shop-simulator.yml:1]: Understood, just wanted to make sure ^^

Comment (SaxbyMod) [reply games/data/tcg-card-shop-simulator.yml]: Hopefully resolved

**Review by anttimaki — COMMENTED** [review]
- Since the CI pipeline can't be run against PRs from forks currently, I created #78 with the same changes
- Note that I fixed the dangling commas in two fields. This doesn't require action from you now, but if you make new PRs in the future, you can run the yml by some online validator to catch these early
- Now I need someone else from Thunderstore to review #78 before I can merge it. Then I can test and approve the mod manager PR.

**Comment by SaxbyMod** [comment]
> * Since the CI pipeline can't be run against PRs from forks currently, I created [TCG Card Shop Simulator #78](https://github.com/thunderstore-io/ecosystem-schema/pull/78) with the same changes
> * Note that I fixed the dangling commas in two fields. This doesn't require action from you now, but if you make new PRs in the future, you can run the yml by some online validator to catch these early
> * Now I need someone else from Thunderstore to review [TCG Card Shop Simulator #78](https://github.com/thunderstore-io/ecosystem-schema/pull/78) before I can merge it. Then I can test and approve the mod manager PR.

Understood, sorry about the hassle, dont know how I forgot to check a schema validator lmao

**Comment by anttimaki** [comment]
Changes merged in #78

**Pull request closed without merging**
